params/bootnodes.go

```diff
@@ -47,6 +53,12 @@ var SepoliaBootnodes = []string{
 	"enode://9e9492e2e8836114cc75f5b929784f4f46c324ad01daf87d956f98b3b6c5fcba95524d6e5cf9861dc96a2c8a171ea7105bb554a197455058de185fa870970c7c@138.68.123.152:30303", // sepolia-bootnode-1-ams3
 }
 
+var OPSepoliaBootnodes = []string{
```

Comment:
We could move this into the superchain registry. Ideally matching the bootnode text file format that eth2 clients use for V5 bootnodes. And ideally with these shared between all superchain networks, since discv5 is not specific per network.

Reply:
discv5 isn't network specific, but I feel like I saw a performance benefit of running it per network. There is a lot I want to do to clean up bootnodes, but we just need something to get snap sync working